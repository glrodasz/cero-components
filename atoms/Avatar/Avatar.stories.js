import { Avatar, styles, options } from '.'

import {
  getTemplate,
  getListTemplate,
  getOptionsArgTypes,
} from '../../helpers/storybook'

const Template = getTemplate(Avatar, styles)
const ListTemplate = getListTemplate(Avatar, styles)

export default {
  title: 'Atoms/Avatar',
  component: Avatar,
  args: {
    src: 'https://picsum.photos/id/1027/200/200',
  },
  argTypes: {
    size: getOptionsArgTypes(options.sizes),
  },
  parameters: { __sb: { fd: 'row' } },
}

export const Default = Template.bind({})

export const Sizes = ListTemplate.bind({})
Sizes.args = {
  items: options.sizes.map((size) => ({ size })),
}
