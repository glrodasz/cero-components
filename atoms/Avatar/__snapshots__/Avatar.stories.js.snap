// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Avatar Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="avatar"
  >
    <picture
      className="picture is-rounded with-border"
    >
      <img
        src="https://picsum.photos/id/1027/200/200"
        style={
          Object {
            "height": 44,
            "maxWidth": 44,
          }
        }
      />
    </picture>
  </div>
</div>
`;

exports[`Storyshots Atoms/Avatar Sizes 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="avatar"
  >
    <picture
      className="picture is-rounded with-border"
    >
      <img
        src="https://picsum.photos/id/1027/200/200"
        style={
          Object {
            "height": 22,
            "maxWidth": 22,
          }
        }
      />
    </picture>
  </div>
  <div
    className="avatar"
  >
    <picture
      className="picture is-rounded with-border"
    >
      <img
        src="https://picsum.photos/id/1027/200/200"
        style={
          Object {
            "height": 44,
            "maxWidth": 44,
          }
        }
      />
    </picture>
  </div>
  <div
    className="avatar"
  >
    <picture
      className="picture is-rounded with-border"
    >
      <img
        src="https://picsum.photos/id/1027/200/200"
        style={
          Object {
            "height": 66,
            "maxWidth": 66,
          }
        }
      />
    </picture>
  </div>
</div>
`;
